src/styles/card.scss

```diff
@@ -0,0 +1,104 @@
+.card {
+  background-color: $white-accent;
+  font-family: "roboto", sans-serif;
```

Comment:
Consider setting a font family for a body tag to get it inherited by other elements
```suggestion

```